.github/workflows/ci.yml

```diff
@@ -123,8 +123,8 @@ jobs:
         uses: actions/cache@v2
         with:
           path: "**/node_modules"
-          key: ${{ runner.os }}-cacheNodeModules14-${{ steps.nodeModulesCacheKey.outputs.value }}
-          restore-keys: ${{ runner.os }}-cacheNodeModules14-
+          key: ${{ runner.os }}-cacheNodeModules18-${{ steps.nodeModulesCacheKey.outputs.value }}
```

Comment:
```suggestion
          key: ${{ runner.os }}-cacheNodeModules16-${{ steps.nodeModulesCacheKey.outputs.value }}
```

Shouldn't this 16?

Reply:
this is just a unique key that can be used to force update the cache, it doesn't have to match the node version.